### PR TITLE
feat(tooltip): [tooltip] add text content wrapper and content-max-height prop

### PR DESCRIPTION
**Pull request by gimmyhehe**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
  - Introduced a new `content-max-height` property for tooltips, allowing configuration of maximum height for content display.
  - Added a demo showcasing the `content-max-height` functionality with a tooltip displaying extensive text.
  
- **Bug Fixes**
  - Enhanced tooltip content selection in tests for better accuracy.

- **Documentation**
  - Updated demo entries to include localized descriptions for new tooltip properties.

- **Style**
  - Added a new CSS class for improved tooltip content styling, ensuring scroll behavior for lengthy content.

- **Tests**
  - New test cases implemented to validate tooltip behavior with dynamic content height settings.

### CODE REVIEW

File: packages/theme/src/tooltip/index.less [hunk 1]
```diff
@@ -26,6 +26,10 @@
     outline-width: 0;
   }
 
+  &__content-wrapper {
```
Comment (zzcr) [inline packages/theme/src/tooltip/index.less:29]: saas主题也需要适配这个样式

File: examples/sites/demos/pc/app/tooltip/content-max-height.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test'
+
+test('内容最大高度', async ({ page }) => {
+  page.on('pageerror', (exception) => expect(exception).toBeNull())
```
Comment (zzcr) [inline examples/sites/demos/pc/app/tooltip/content-max-height.spec.js:4]: 标题改下，触发下E2E测试用例哈

File: packages/theme/src/tooltip/index.less (hunk 1 above)
Comment (gimmyhehe) [reply packages/theme/src/tooltip/index.less:29]: 已添加

File: examples/sites/demos/pc/app/tooltip/content-max-height.spec.js (hunk 2 above)
Comment (gimmyhehe) [reply examples/sites/demos/pc/app/tooltip/content-max-height.spec.js:4]: 已添加

**Pull request merged**